### DISCUSSION TITLE
Z axis rotation capability?

### DISCUSSION THREAD

**Discussion by NovelSora** [AnimeEffectsDevs/AnimeEffects#43] [Ideas]
I don't know if its actually possible to code this into the program _(as in if its support 3D capability in the slightest)_, but its something I'd love to see _(even if I haft to potentially code it myself which I am fine with)_. Unless there is a trick or something I have missed to create a similar effect

**Comment by p-yukusai** [comment]
Holy hell I'm sorry for the late reply, i apparently do not get notifications for this.
You're always free to take a look at the renderer code to see the technical feasibility of 3d-fying AnimeEffects, from what i remember it will be quite the tricky thing to implement but if you wanna go for it you're more than welcome to.
The latest code is in my own fork of AE on the qt6 branch if you wanna clone it
